helsingborg-stad/app-mitt-helsingborg
# Refactor/remove unused file uploader components

## Explain the changes you’ve made
Remove unused file uploader components because they have been replaced with filepicker.

## Explain why these changes are made
We only need one component for uploading files, hence removing the others.

## Explain your solution
Remove references and code to the other components.

## How to test

1. Checkout this branch
2. Fire upp the simulator by running the command `yarn ios` or `yarn android`
3. Open any form where the user should upload files
4. The form should be working as usual 

## Tested environments

- [] Storybook on a iOS device/simulator.
- [] Building the Application on a Android device/simulator.
